webrtc.html

```diff
@@ -4790,23 +4770,112 @@ <h2>Methods</h2>
+                      is not undefined, set <var>expires</var> to
+                      <var>certificateExpiration</var>.<code>expires</code>.</p>
+                    </li>
+                  </ol>
+                </li>
+                <li>
+                  <p>Let <var>normalizedKeygenAlgorithm</var> be the result of
+                  <a data-cite="!WebCryptoAPI#dfn-normalize-an-algorithm">normalizing an algorithm</a>
+                  with an operation name of <code>generateKey</code> and a
+                  <a data-cite="!WebCryptoAPI#dfn-supportedAlgorithms">supportedAlgorithms</a>
+                  value specific to production of certificates for
+                  <code>RTCPeerConnection</code>.</p>
+                </li>
+                <li>
+                  <p>If the above normalization step fails with an <var>error</var>,
+                  return a promise that is <a>rejected</a> with <var>error</var>.</p>
+                </li>
+                <li>
+                  <p>If the <var>normalizedKeygenAlgorithm</var> parameter
+                  identifies an algorithm that the <a>user agent</a> cannot
+                  or will not use to generate a certificate for
+                  <code>RTCPeerConnection</code>, return a promise that is
+                  <a>rejected</a> with a <code>DOMException</code> of type
+                  <code>NotSupportedError</code>. In particular,
+                  <var>normalizedKeygenAlgorithm</var> MUST be an asymmetric
+                  algorithm that can be used to produce a signature used to
+                  authenticate DTLS connections.</p>
+                </li>
+                <li>
+                  <p>Let <var>p</var> be a new promise.</p>
+                </li>
+                <li>
+                  <p>Run the following steps in parallel:</p>
+                  <ol>
+                    <li>
+                      <p>Perform the generate key operation specified by
+                      <var>normalizedKeygenAlgorithm</var> using
+                      <var>keygenAlgorithm</var>.</p>
+                    </li>
+                    <li>
+                      <p>Let <var>generatedKeyingMaterial</var> and
+                      <var>generatedKeyCertificate</var> be the private
+                      keying material and certificate generated by the
+                      above step.</p>
+                    </li>
+                    <li>
+                      <p>Let <var>certificate</var> be a new
+                      <code><a>RTCCertificate</a></code> object.</p>
+                    </li>
+                    <li>
+                      <p>Set <var>certificate</var>.[[\Expires]] to the
+                      current time plus <var>expires</var> value. A <a>user
+                      agent</a> MAY choose to limit the value of the
+                      <var>certificate</var>.[[\Expires]] value.</p>
```

Comment:
This MAY is lost. Intentional? Without this, JS could create a certificate that never expires. Is that wise?

Reply:
I do not think the PR is changing anything there.
As discussed elsewhere, the idea is to strengthen this MAY, given several implementations seem to cap by 1 year.